docs/index.rst

```diff
@@ -4,6 +4,8 @@ Welcome to Horovod's documentation!
 .. toctree::
    :maxdepth: 2
 
+   readme_include
```

Comment:
I think the first page needs to have a distillation of the info from our readme, with links to find out more information on the sub-pages.  Bloated README needs to go.

Reply:
I agree. Maybe the Why not traditional Distributed TensorFlow?, Keras, Estimator API, MXNet, PyTorch, mpi4py sections can be similar to the Inference section, with 1-2 sentences and then link to a sub-page

Reply:
Sounds good! I'll implement it as soon as we agree about which parts should be subpages

Reply:
Let's start with Stephanie's suggestion.